bodhi/client/bindings.py

```diff
@@ -797,7 +797,7 @@ def update_str(self, update, minimal=False):
             ]
 
         if update['notes']:
-            buf = six.moves.cStringIO(update['notes'])
+            buf = six.moves.cStringIO(update['notes'].encode('utf-8'))
```

Comment:
So I think there are a few tricky things going on in here, and after playing around a bit with the modified test I mentioned in my other comment I haven't arrived at a good solution that works in Python 2 and Python 3 at the same time.

One tricky thing is that ```cStringIO``` has different behavior here than ```StringIO``` has. Note also that Python 3 doesn't have a ```cStringIO``` and so this ```six.moves``` thing is just an alias for ```io.StringIO``` in Python 3.

We may need to do some "If Python 2... else..." type things to solve this one.

Reply:
I think we don't need to use cStringIO at all... looking how updates['comments'] are managed I removed those six stuff and it seems to work just fine in Python 2 and Python 3. Correct me if there was some reason to use cStringIO that I didn't see.

Reply:
Yeah I think that makes a lot of sense, thanks!